docs/general/installation/community.md

```diff
@@ -0,0 +1,139 @@
+---
+uid: installation-community
+title: Community Maintained Packages
+description: Community maintained packages.
+sidebar_position: 5
+---
+
+# Community Maintained Packages
+
+These packages are maintained by community packagers independent of Jellyfin for distributions not officially supported by Jellyfin. For these distributions, [container images](/docs/general/installation/container) are recommended.
```

Comment:
>For these distributions, [container images](/docs/general/installation/container) are recommended.

This is where we get into where our line should be. Personally I think it should be a little less forceful, something like.

>We can't always support problems with community-maintained packages; if you encounter trouble with them, please contact their maintainers first. If you're not sure, we recommend using [container images](/docs/general/installation/container) on these systems instead.